packages/web-client/tests/acceptance/persistence-view-restore-test.ts

```diff
@@ -134,9 +134,18 @@ module('Acceptance | persistence view and restore', function () {
       });
 
       workflowPersistenceService.persistData('canceled', {
-        name: 'PREPAID_CARD_ISSUANCE',
+        name: `PREPAID_CARD_ISSUANCE`,
         state: buildState({
           meta: {
+            completedCardNames: [
```

Comment:
Workflows can get canceled in the middle of the progress and their whole state will get persisted.